UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.FulfilmentService/Services/FulfilmentDataService.cs

```diff
@@ -74,5 +74,20 @@ public async Task QueryAndDownloadFileShareServiceFiles(SalesCatalogueServiceRes
                 logger.LogInformation(EventIds.DownloadFileShareServiceFilesCompleted.ToEventId(), "Download File share service request completed for {BatchId}", message.BatchId);
             }
         }
+        private async Task CreateAncillaryFiles(string batchId, string exchangeSetRootPath)
+        {
+            await DownloadReadMeFile(batchId, exchangeSetRootPath);
+        }
+
+        public async Task DownloadReadMeFile(string batchId, string exchangeSetRootPath)
+        {
+            logger.LogInformation(EventIds.SearchDownloadReadMeFileRequestStart.ToEventId(), "Search and download ReadMe Text File start for {BatchId}", batchId);
```

Comment:
Can we log Search and Download ReadMeFileRequestStart and ReadMeFileRequestCompleted separately.

Reply:
Done